package.json

```diff
@@ -139,20 +139,20 @@
     "@cliqz/adblocker-webextension-cosmetics": "^1.20.0",
     "@cliqz/url-parser": "^1.1.4",
     "abortcontroller-polyfill": "^1.5.0",
-    "anonymous-credentials": "https://github.com/cliqz-oss/anonymous-credentials/releases/download/1.0.0/anonymous-credentials-1.0.0.tgz",
+    "anonymous-credentials": "https://github.com/whotracksme/anonymous-credentials/releases/download/1.0.0/anonymous-credentials-1.0.0.tgz",
     "fast-deep-equal": "^3.1.3",
     "jsonschema": "^1.4.0",
-    "math-expression-evaluator": "^1.3.4",
+    "math-expression-evaluator": "^1.3.8",
     "moment": "^2.29.1",
     "node-fetch": "^2.6.1",
     "node-forge": "^0.10.0",
-    "pako": "^2.0.2",
+    "pako": "^2.0.4",
     "punycode": "^2.1.1",
-    "rusha": "^0.8.13",
-    "rxjs": "^6.6.3",
+    "rusha": "^0.8.14",
+    "rxjs": "^7.5.2",
```

Comment:
this may be a breaking change for anti-tracking. please revert to 6.x

Reply:
In the current state, the PR introduces breakage. That is why I changed to draft. Perhaps it would be wise to upgrade in independent steps. Some updates also break the build system.